docs/user-guide/src/bmo/bmo.md

```diff
@@ -0,0 +1,98 @@
+<h1>Bare Metal Operator</h1>
+
+The Bare Metal Operator (BMO) is a custom Kubernetes controller that deploys baremetal hosts, represented by BMH, as Kubernetes nodes. To this end Ironic is used.
+
+BMO can also work with the Cluster API Provider Metal3 (CAPM3) controller. With the involvment of CAPM3 and Ironic, a simplified information flow path is shown below:
```

Comment:
Should this be moved down below the BMH example?

Reply:
Thanks, moved.